docs/src/main/sphinx/client/cli.rst

```diff
@@ -285,7 +285,21 @@ and prompts the CLI for your password:
 
 .. code-block:: text
 
-  ./trino --server https://trino.example.com --user=myusername --password
+  ./trino --server https://trino.example.com --user=exampleusername --password
+
+Alternatively, set the password as the value of the ``TRINO_PASSWORD``
+environment variables. Typically use quotes to avoid problems with special
```

Comment:
I suggest to be explicit about using single quotes and not double quotes.
```suggestion
environment variables. Typically use single quotes to avoid problems with special
```

Reply:
Hahah .. fair .. although I think "" would work too .. but thats nuances .. not to worry about